pandas/tests/frame/methods/test_convert_dtypes.py

```diff
@@ -35,6 +35,27 @@ def test_convert_empty(self):
         empty_df = pd.DataFrame()
         tm.assert_frame_equal(empty_df, empty_df.convert_dtypes())
 
+    def test_convert_empty_categorical_to_pyarrow(self):
```

Comment:
```suggestion
    def test_convert_empty_categorical_to_pyarrow(self):
        pytest.importorskip("pyarrow")
```

Reply:
Use the `@td.skip_if_no` decorator instead.